tests/test_grid.py

```diff
@@ -41,7 +41,7 @@ def setUp(self):
         Create a test non-toroidal grid and populate it with Mock Agents
         """
         width = 3  # width of grid
-        height = 5  # height of grid
+        height = 6  # height of grid
```

Comment:
Needs to document that the height has to be even here.

Reply:
done